main.tf

```diff
@@ -75,6 +78,19 @@ data "template_file" "this" {
   }
 }
 
+data "template_cloudinit_config" "this" {
```

Comment:
I was confused until I saw that this, by default, renders it as base64 encoded https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/cloudinit_config#base64_encode

Reply:
This is not an issue, I just wanted to point this out in case it confuses someone else